src/2018/transitioning/modules/macros.md

```diff
@@ -1,9 +1,7 @@
 # Macro changes
 
-Now that `extern crate` is gone, there's a problem: to import macros, you
-need to put `#[macro_use]` on the `extern crate` declaration.
-
-To fix this issue, you can use `use` to import macros from extern crates.
+In Rust 2018, you can use `use` to import specific macros from extern crates,
```

Comment:
The word "use" is repeated twice here, which feels a bit sub-optimal. Not sure what to replace it with tho.

Reply:
Replace `extern crates` with `external crates` :)

Reply:
What about:

In Rust 2018, the `use` keyword allows the importing of specific macros from external crates,

Reply:
If we reorder:
> In Rust 2018, specific macros from external crates can be imported with `use`.